eclass/secureboot.eclass

```diff
@@ -101,13 +101,15 @@ secureboot_pkg_setup() {
 # @USAGE: <input file> <output file>
```

Comment:
You want to update this too.

Also the commit message summary is too long.

Reply:
Would `[<output file>]` be the correct syntax here or `[output file]`?

Reply:
I'd use the former.

Reply:
Alright, Fixed.